sshd/issue_invite.sh

```diff
@@ -9,19 +9,22 @@ set -e
 
 INVITE_CODE=
 USERNAME=getter
+COMPLETE=false
 
 function usage () {
-  echo "$0 [-u username] [-i invite code]"
+  echo "$0 [-u username] [-i invite code] [-c]"
   echo "  -i: invite code (if unspecified, a new invite code is generated)"
   echo "  -u: username (default: getter)"
+  echo "  -c: output complete invite URL params (for use with cloud)"
```

Comment:
This script is only used for cloud...it might better to say "for manual installs".